vivisect/__init__.py

```diff
@@ -1076,7 +1081,8 @@ def makeOpcode(self, va, op=None, arch=envi.ARCH_DEFAULT):
                     refva, refsize, reftype, refinfo = self.getLocation(xrfrom)
                     if reftype != vivisect.LOC_OP:
                         continue
-
+                    if refva == op.va:
```

Comment:
this would be a great place for a comment.

`# if this "other xref" is actually, this one, skip any changes and move on.`

Reply:
:+1:

Reply:
that's so totally not what i read as going on  :)
this is iterating through the xrefs to a given rdest.
one of those will be *this* one.  if so, don't break it up.

that's what i read.  if that's not correct, i'm *really* glad you're making a comment here.

Reply:
I actually got a bit ahead of myself. Still a bit scatter brained today. You're right, It should be more like:

`If we've already constructed this opcode location and made the xref to the new codeblock, that should mean we've already made the jump table, so there should be no need to split this jump table.`

I'll change the wording to that ^^